datadog/resource_datadog_dashboard.go

```diff
@@ -4917,6 +4933,156 @@ func buildTerraformServiceLevelObjectiveDefinition(datadogDefinition datadogV1.S
 	return terraformDefinition
 }
 
+//
+// SLO (Service Level Objective) List Widget Definition helpers
+//
+
+func getSloListDefinitionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"request": {
+			Description: "A nested block describing the request to use when displaying the widget. Exactly one `request` block is allowed.",
+			Type:        schema.TypeList,
+			Required:    true,
+			MaxItems:    1,
+			MinItems:    1,
+			Elem: &schema.Resource{
+				Schema: getSloListRequestSchema(),
+			},
+		},
+		"title": {
+			Description: "The title of the widget.",
+			Type:        schema.TypeString,
+			Optional:    true,
+		},
+		"title_size": {
+			Description: "The size of the widget's title (defaults to 16).",
+			Type:        schema.TypeString,
+			Optional:    true,
+		},
+		"title_align": {
+			Description:      "The alignment of the widget's title.",
+			Type:             schema.TypeString,
+			ValidateDiagFunc: validators.ValidateEnumValue(datadogV1.NewWidgetTextAlignFromValue),
+			Optional:         true,
+		},
```

Comment:
in doc page for APIs, seems `type` is a required field for slo list widget:  https://docs.datadoghq.com/dashboards/widgets/slo_list/#api  
don't we need the `type` field here as well?

Reply:
Nah, there is only one valid value for `type` anyways (`slo_list`), so having the ability to configure it would only introduce chance for error

Reply:
For context, our API uses `type` to know which type of widget you're trying to create. We don't need that here because we already know what type of widget you're trying to create, so we automatically fill that in for you

Reply:
Actually, following that same logic I should remove `request_type` as well because there is only one valid value. Making the user specify it is unnecessary. I'll make that update

Reply:
all good! yeah, I also confirmed datadogV1.NewSLOListWidgetRequestWithDefaults() in go client that it does initialize `type` for slo list definition :)

Reply:
> Actually, following that same logic I should remove `request_type` as well because there is only one valid value. Making the user specify it is unnecessary. I'll make that update

I looked into this and turns out it's normal for this to be the case, so I'm going to leave it as-is for now